src/frontend/src/components/base/Accordion/AccordionItem.tsx

```diff
@@ -26,12 +27,20 @@ export const AccordionItem = ({
     isMobile,
     overridePanelOpen,
     children,
+    lazyLoad,
     ...props
 }: AccordionItemProps) => {
     const styles = accordionStyles({ overridePanelOpen: overridePanelOpen });
+    const [isLazyLoaded, setIsLazyLoaded] = useState(false);
 
     return (
-        <Disclosure id={props.id} className={styles.accordionItemWrapper()}>
+        <Disclosure
+            id={props.id}
+            className={styles.accordionItemWrapper()}
+            onExpandedChange={isExpanded =>
+                lazyLoad && isExpanded && !isLazyLoaded && setIsLazyLoaded(true)
```

Comment:
It took me a bit to figure out why we have both `lazyLoad` and `isLazyLoaded`, and also what they mean here in this line. It might be helpful to add some quick comments about them.